src/styles/blocks/card.css

```diff
@@ -0,0 +1,106 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  background: #ffff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  max-width: 200px;
+  padding: 32px 16px 16px ;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  align-self: center;
+  padding-bottom: 40px;
+}
+
+.card__name {
+  width: 166px;
+  height: 36px;
+  font-style: normal;
+  font-weight: var(--font-weight-body-m);
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-text-main);
+  padding-bottom: 4px;
+}
+
+.card__code {
+  font-style: normal;
+  font-weight: var(--font-weight-body-s);
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-text-secondary);
+  padding-bottom: 16px;
+}
+
+.card__rate {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__reviews {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: right;
+  font-style: normal;
+  font-weight: var(--font-weight-body-s);
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-text-main);
+}
+
+.card__price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  font-style: normal;
+  font-weight: var(--font-weight-body-s);
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-text-secondary);
+}
+
+.card__price-amount {
+  font-style: normal;
+  font-weight: var(--font-weight-body-l);
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--color-text-main);
+}
+
+.card__buy {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 166px;
+  height: 40px;
```

Comment:
no needs to hardcode width/height like this for elements with content inside. Use padding instead